### PR TITLE
feat(request withdrawal) Added progress bar

### PR DESCRIPTION
**Pull request by chrismaree**
This small PR adds a little feature to the request withdrawal page to show the progress of a withdrawal request.

image

### CODE REVIEW

File: features/manage-position/Withdraw.tsx [hunk 1]
```diff
@@ -123,6 +124,10 @@ const Deposit = () => {
     );
     const hasPendingWithdraw = pendingWithdraw === "Yes";
     const pendingWithdrawTimeRemaining = withdrawPassTime - Number(currentTime);
+    const progressBarPercent =
```
Comment (nicholaspai) [inline features/manage-position/Withdraw.tsx:127]: What if `withdrawlLiveness` is 0 will this crash?

Comment (chrismaree) [reply features/manage-position/Withdraw.tsx:127]: it doesent crash it but I agree it should not calculate this if it is zero. Have updated PTAL.

**Comment by chrismaree** [comment]
@nicholaspai ptal at this. small pr that we should be able to get in.

**Review by nicholaspai — APPROVED** [review]
LGTM

**Pull request merged**